### PR TITLE
Basic post-processing and analysis service with kafka

### PR DESCRIPTION
**Pull request by rasmitha-koduru**

### CODE REVIEW

File: model_execution/producer.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from time import sleep
```
Comment (adithyaselv) [inline model_execution/producer.py]: Producer shouldn't be separate for now . a single py file will both produce and consume .

File: model_execution/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+Model_execution service:
+
+Requirements:
+
+1. To Run Kafka follow instructions from Readme file from Kafka.
+
+2. Python 3 : [https://www.python.org/downloads/]
+
+3. kafka-python pip install kafka-python and pymongo from pip install pymongo
```
Comment (adithyaselv) [inline model_execution/README.md]: annotate code

File: model_execution/producer.py (hunk 1 above)
Comment (rasmitha-koduru) [reply model_execution/producer.py]: We will not be having this produce file as we improve this service so i added it separately to test. Should i change it to a single file?

Comment (adithyaselv) [reply model_execution/producer.py]: if this is only for testing can remove the file before merging this PR to master

**Comment by adithyaselv** [comment]
Also isn't this Post processing service ? please rename the PR

File: model_execution/README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+```
+docker pull mongo:4.0.4
+```
+
+
+Running:
+```
+python model_execution.py
+```
+#Producer is just for test purpose will be removed as service evolves.
```
Comment (adithyaselv) [inline model_execution/README.md]: 👍

**Review by adithyaselv — APPROVED** [review]
good

**Pull request merged**